src/config/config.php

```diff
@@ -54,6 +54,7 @@
         //'fil'         => ['name' => 'Filipino',               'script' => 'Latn', 'native' => 'Filipino', 'regional' => 'fil_PH'],
         //'fr'          => ['name' => 'French',                 'script' => 'Latn', 'native' => 'français', 'regional' => 'fr_FR'],
         //'fr-CA'       => ['name' => 'Canadian French',        'script' => 'Latn', 'native' => 'français canadien', 'regional' => 'fr_CA'],
+        //'en-CA'       => ['name' => 'Canadian English',       'script' => 'Latn', 'native' => 'Canadian English', 'regional' => 'en_CA'],
```

Comment:
The languages in this file are sorted as "English, Canada", rather than their display name of "Candian English".

This is so that all the "Englishes" are shown together (under "E") - which is where a user would look for them in an alphabetical list.   Can you move it?  Thanks.

Reply:
Agreed 😄

Reply:
I have updated the PR